packages/components/switch/src/switch.ts

```diff
@@ -169,6 +169,13 @@ export const switchProps = buildProps({
     type: [Boolean, String, Number],
     default: false,
   },
+  /**
+   * @description native input aria-label
+   */
+  label: {
+    type: String,
+    default: 'Switch',
```

Comment:
```suggestion
    default: undefined,
```

There should not be a default value here, as I found it would affect the screen reader's prompts in the form